Documentation Update (#358)
spica-engine/spica: docs/content/guide/passport.md

## Passport

Fundamentally, this module is used for authentication and authorization on Spica Client and API.

### Authentication

> Out of the box, Spica supports two different authentication stretagies. This document will explain the fundemantals of those stretegies.

Reviewer: ```suggestion
Out of the box, Spica supports two different authentication strategies. This document will explain the fundemantals of those stretegies.
```

#### Identity

> Fundemantally, Identities are the users of a Spica instance. It contains **identified** and **password** informations for login.

Reviewer: ```suggestion
Fundamentally, Identities are the users of a Spica instance. It contains **identifier** and **password** informations for authorization.
```

> Right after the installation, Spica creates a default identitiy to get you started. We suggest you to change at least its password before going live.

> To create an Identitiy, navigate to **System** -> **Identities** in the left-hand menu.

Reviewer: ```suggestion
To create an Identitiy, navigate to **System** -> **Identities** in the left-side menu.
```

- Click "+" icon on the top right toolbar.
- Fill the **Identifier** and **Password** field.
- Press **Save**

##### Adding Additional Properties

> If you want to store additional informations on **Identities**, you can create custom fields. To create a custom field on Identities, navigate to **System** -> **Settings** in the left-hand menu.

Reviewer: ```suggestion
If you want to store additional information on **Identities**, you can create custom fields. To create a custom field on Identities, navigate to **System** -> **Settings** in the left-hand menu.
```

#### API Key

> Instead of Identity, API key is allows the machine to machine communication. The token, it provides doesn't have an expiration date so it can be used as long as it's intentionally deleted from Spica.

Reviewer: ```suggestion
Instead of Identity, the API key allows the machine to machine communication. The token, it provides doesn't have an expiration date so it can be used as long as it is not intentionally deleted from Spica.
```

To create an API Key, navigate to **System** -> **API Keys** in the left-hand menu.

- Click "+" icon on the top right toolbar.
- Fill the **name** and **description** field.
- Press **Save**

Once the saving completed, you can use the **API Key** value on your API calls on header section.

> For more informations about how to use API Key, please refer the API Documentations.

<!-- #### SSO Integration -->

### Policies

> Policies are basically a set of rules which can be attached to your **Identities** and **API Keys** to encapsulate their behavior.

Reviewer: ```suggestion
Policies are basically a set of rules which can be attached to your **Identities** and **API Keys** to encapsulate and control their capabilities.
```
